board/stm32f4_config.h

```diff
@@ -0,0 +1,34 @@
+#define PANDA
+#include "stm32f4xx.h"
+#include "stm32f4xx_hal_gpio_ex.h"
+
+#define CORE_FREQ 96U // 96Mhz
+//APB1 - 48Mhz, APB2 - 96Mhz
+#define APB1_FREQ CORE_FREQ/2U 
+#define APB2_FREQ CORE_FREQ/1U
+
+#define BOOTLOADER_ADDRESS 0x1FFF0004
+#define MCU_IDCODE 0x463U
+
+// Around (1Mbps / 8 bits/byte / 12 bytes per message)
+#define CAN_INTERRUPT_RATE 12000U
+
+#define MAX_LED_FADE 8192U
+
+#define NUM_INTERRUPTS 102U                // There are 102 external interrupt sources (see stm32f413.h)
+
+#define HEARTBEAT_TIMER_IRQ TIM1_BRK_TIM9_IRQn
+#define HEARTBEAT_TIMER TIM9
+
+#define MICROSECOND_TIMER TIM2
+
+#define PROVISION_CHUNK_ADDRESS 0x1FFF79E0
+#define SERIAL_NUMBER_ADDRESS 0x1FFF79C0
+
+void early_gpio_float(void) {
```

Comment:
a lot of this is shared with the f2, should we move some of it to a common header?

Reply:
Agree, will do